packages/rancher-logging/generated-changes/patch/templates/_helpers.tpl.patch

```diff
@@ -93,4 +93,12 @@
 +    Time_Key          timestamp
 +    Time_Format       %FT%H:%M:%S.%L
 +{{- end -}}
-\ No newline at end of file
++
++{{/*
++Set kubernetes log options if they are configured
++*/}}
++{{- define "requireFilterKubernetes" -}}
++{{- if or .Values.fluentbit.filterKubernetes.Merge_Log .Values.fluentbit.filterKubernetes.Merge_Log_Key .Values.fluentbit.filterKubernetes.Merge_Trim .Values.fluentbit.filterKubernetes.Merge_Parser}}
```

Comment:
Looks like we either missed this or it's new, but the end of the line sprig formatter should have a whitespace and a dash to ensure we don't add blank newlines.
```
Merge_Parser -}}
```